tests/jobs/test_job_decorator.py

```diff
@@ -956,7 +963,9 @@ async def execute_default(self) -> bool:
             return True
 
     test1 = TestClass(coresys)
+    # pylint: disable=protected-access
     client = coresys.homeassistant.websocket._client
+    # pylint: enable=protected-access
```

Comment:
This could be a single line

```suggestion
    # pylint: disable-next=protected-access
    client = coresys.homeassistant.websocket._client
```

Reply:
I did not know about `disable-next`. I will definitely remember that as it helps in a lot of other places too 👍